[2022-03-29 00:11:01,962] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_cine scheduled__2022-03-28T05:10:41.508357+00:00 [queued]>
[2022-03-29 00:11:01,987] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_cine scheduled__2022-03-28T05:10:41.508357+00:00 [queued]>
[2022-03-29 00:11:01,987] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-29 00:11:01,987] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-29 00:11:01,987] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-29 00:11:02,467] {taskinstance.py:1264} INFO - Executing <Task(_PythonDecoratedOperator): get_data_cine> on 2022-03-28 05:10:41.508357+00:00
[2022-03-29 00:11:02,470] {standard_task_runner.py:52} INFO - Started process 172 to run task
[2022-03-29 00:11:02,475] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_dag', 'get_data_cine', 'scheduled__2022-03-28T05:10:41.508357+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/postgres_conn.py', '--cfg-path', '/tmp/tmp3f04qu4d', '--error-file', '/tmp/tmpo3bqn4k5']
[2022-03-29 00:11:02,475] {standard_task_runner.py:77} INFO - Job 183: Subtask get_data_cine
[2022-03-29 00:11:02,874] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_dag.get_data_cine scheduled__2022-03-28T05:10:41.508357+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-03-29 00:11:03,866] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=postgres_dag
AIRFLOW_CTX_TASK_ID=get_data_cine
AIRFLOW_CTX_EXECUTION_DATE=2022-03-28T05:10:41.508357+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-28T05:10:41.508357+00:00
[2022-03-29 00:11:03,921] {base.py:70} INFO - Using connection to: id: airflow-universities. Host: training-main.cghe7e6sfljt.us-east-1.rds.amazonaws.com, Port: 5432, Schema: training, Login: alkymer, Password: ***, extra: {}
[2022-03-29 00:11:05,397] {python.py:175} INFO - Done. Returned value was: None
[2022-03-29 00:11:05,408] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=postgres_dag, task_id=get_data_cine, execution_date=20220328T051041, start_date=20220329T051101, end_date=20220329T051105
[2022-03-29 00:11:06,270] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-29 00:11:06,738] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
